src/main/resources/main.js

```diff
@@ -19,3 +21,16 @@ eventLib.listener({
 if (licenseManagerLib.isCurrentLicenseValid()) {
     licenseManagerLib.activateLicense();
 }
+
+contextLib.run({
```

Comment:
on clustered environments this code could run into race condition issues.
It should be wrapped into `isMaster` check https://developer.enonic.com/docs/xp/stable/api/lib-cluster#ismaster